### DISCUSSION TITLE
Add support for Milvus

### DISCUSSION THREAD

**Discussion by hegdeashwin** [langflow-ai/langflow#2093] [Ideas]
I'm using Milvus as Vector store which is kind of missing here.
Not sure if anyone working on this, Would like to contribute to build the Milvus component to the store.

Currently going over codebase. Is there any channel where we can reach out for guidance in case required?

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
